src/EditPokemon/EditPokemon.js

```diff
@@ -27,7 +27,11 @@ export default function EditPokemon(props) {
                     defaultValue={selectedPokemon.name}
                 ></input>
 
-                <select name="type" onChange={updateCardData}>
+                <select
+                    value={selectedPokemon.type.id}
```

Comment:
this is a super nit pick but I might move these attributes around to be alphabetical